Kernel/kprintf.cpp

```diff
@@ -23,19 +23,19 @@ namespace Kernel {
 extern Atomic<Graphics::Console*> g_boot_console;
 }
 
-static bool serial_debug;
+static bool s_serial_debug_enabled;
 // A recursive spinlock allows us to keep writing in the case where a
 // page fault happens in the middle of a dbgln(), etc
 static RecursiveSpinlock s_log_lock { LockRank::None };
 
-void set_serial_debug(bool on_or_off)
+void set_serial_debug_enabled(bool desired_state)
 {
-    serial_debug = on_or_off;
+    s_serial_debug_enabled = desired_state;
 }
 
-int get_serial_debug()
+bool get_serial_debug_enabled()
```

Comment:
```suggestion
bool is_serial_debug_enabled()
```
I think `is_` makes more sense here and seems how methods like this are named elsewhere.

Reply:
that makes sense, I will change it when I get off work